examples/reduce/reduce.jl

```diff
@@ -12,7 +12,7 @@ using CUDAdrv, CUDAnative
 #
 
 # Reduce a value across a warp
-@inline function reduce_warp{F<:Function,T}(op::F, val::T)::T
+@inline function reduce_warp(op::Function, val::T)::T where {T}
```

Comment:
I don't think it matters: https://docs.julialang.org/en/latest/manual/style-guide.html#Don%27t-use-unnecessary-static-parameters-1